backend/e4h-services/ingestion-service/app/utils/excel_utils.py

```
@@ -172,10 +206,118 @@ def add_validations_to_excel(file_path: str, sheet_name: str, validations: Dict[
+    """
+    Add non-blank validations to an Excel sheet given file path and sheet name.
+
+    :param file_path: Path to the Excel file
+    :param sheet_name: Name of the worksheet where validations should be applied
+    :param facility_schema: List of dicts with column definitions
+    :param allow_blank_map: Dict {header_name: bool} to override allow_blank
+    """
+    try:
+        wb = load_workbook(file_path)
+        if sheet_name not in wb.sheetnames:
+            raise ValueError(f"Sheet '{sheet_name}' not found in {file_path}")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)